packages/components/pagination/src/pagination.tsx

```diff
@@ -32,6 +32,8 @@ interface PaginationProps {
   onPageChange?: (page: number) => void
   onItemPerPageChange?: (page: number) => void
   recordName?: string
+  hideOnSinglePage?: boolean
```

Comment:
This prop doesn't make sense. Please use a condition outside the component to check if we should render component or not.